vslib/VirtualSwitchSaiInterface.cpp

```diff
@@ -981,6 +981,8 @@ sai_status_t VirtualSwitchSaiInterface::queryStatsCapability(
 {
     SWSS_LOG_ENTER();
 
+    uint32_t i=0;
```

Comment:
declare that internally in each for loop defininition, declaring it on function beign is old style C